security-analysis/security-analysis-api/src/main/java/com/powsybl/security/LimitViolation.java

```diff
@@ -211,4 +213,16 @@ public double getValue() {
             return null;
         }
     }
+
+    public String toString() {
+        String base = "Subject id: " + this.subjectId + ", Subject name: " + this.subjectName + ", limitType: " +
+                this.limitType.toString() + ", limit: " + this.limit + ", limitName: " + this.limitName +
```

Comment:
There is no need to call toString. If limitType is null, you would have a NullPointerException whereas it works perfectly without it